app/styles/home.css

```diff
@@ -4,11 +4,18 @@
 }
 
 #home .course {
-  margin-left: 15px;
+  margin: 0 0 15px 15px;
+  padding: 15px 15px 15px 15px;
```

Comment:
also, you should use a proportional unit, like `vmin` ([ref](https://www.w3.org/Style/Examples/007/units.fr.html))